### ISSUE TITLE
Incorrect min_wal_size displayed for PG ver >= 10

### ISSUE THREAD

**Issue by wozniakmateusz** [rapidloop/pgmetrics#38]
In 1.10.0, displaying `max_wal_size` in text reports for PG versions >= 10 has been fixed (https://github.com/rapidloop/pgmetrics/commit/513dcc0e2a16acac3890bee1d24ff20106877340). However, this has not been done for `min_wal_size`, which leads to outputs like:
```
    +--------------------+----------------+
    |            Setting |          Value |
    +--------------------+----------------+
    |          wal_level |        replica |
    |    archive_timeout |              0 |
    |    wal_compression |            off |
    |       max_wal_size | 10240 (10 GiB) |
    |       min_wal_size |  1024 (16 GiB) |
```

**Comment by mdevan** [comment]
Oops. Thanks for reporting. Yes, it is a bug. Will fix soon.

**Comment by mdevan** [comment]
Fixed in 1d59cff.
